src/app/clients/page.tsx

```diff
@@ -1,23 +1,16 @@
-import supabase, { Schema } from "@/supabase";
 import { Metadata } from "next";
 import React from "react";
-
-const dataFetch: () => Promise<Schema["clients"][]> = async () => {
-    const response = await supabase.from("clients").select();
-    return response.data ?? [];
-};
+import { ClientsTableRow, getClientsTableData } from "@/app/clients/getClientsTableData";
+import ClientsPage from "@/app/clients/ClientsPage";
+import Title from "@/components/Title/Title";
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data = await dataFetch();
+    const data: ClientsTableRow[] = await getClientsTableData();
```

Comment:
you probably don't need a type annotation here

Reply:
Yeah, I've removed it now